Frameworks/UIKit/UILabel.mm

```diff
@@ -188,10 +188,6 @@ - (void)_initUILabel {
     WXCTextBlock* textBlock = nil;
```

Comment:
how much closer could it be? :)

---
In reply to: [108989011](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108989011) [](ancestors = 108989011)